content/guides/building/get-source-git.md

```diff
@@ -48,64 +48,75 @@ to the last 10 commits, for example.
 
 ```sh
 git config core.precomposeunicode true
 ```
 
 <h4>Setup an account on Gerrit</h4>
 
-<p>Log in to <a href="https://review.haiku-os.org">Gerrit code review</a>. You currently need a <a href="https://github.com">Github</a> account for logging in.</p>
+<p>Log in to <a href="https://review.haiku-os.org">Gerrit code review</a>. For the time being, using a <a href="https://github.com">GitHub account</a> is necessary.</p>
 
 <div class="alert alert-warning">
-Make sure your ssh key is generated with ed25519. RSA keys will not work with Gerrit.
+Make sure your SSH key is generated using <code class="code">ed25519</code>. RSA keys will <b>not</b> work with Gerrit.
 </div>
-<p>Upload your SSH public key in <a href="https://review.haiku-os.org/settings/#SSHKeys">Gerrit SSH keys settings</a> page. If you don't have a key yet, you can generate one using ssh-keygen.</p>
+<p>Upload your SSH public key in the <a href="https://review.haiku-os.org/settings/#SSHKeys">Gerrit SSH Keys settings</a> page. If you don't have a key yet, you can generate one using ssh-keygen.</p>
 <p>If the e-mail address used in your commits does not match the one in your github account, you will need to add and verify it in <a href="https://review.haiku-os.org/settings/#EmailAddresses">Gerrit E-Mail address settings</a>.</p>
 
-<h4>Build Tools:</h4>
+<div class="alert alert-warning">
+You may need to replace the <code class="varname">$USER</code> parameter in the following commands.
+</div>
+<p>This can happen if the username on your Gerrit account does not match the username that you use to log in to your computer. This would be the case in Haiku itself, as the default username is <code class="code">user</code>. You can make your life easier by <a href="https://linux.die.net/man/5/ssh_config">modifying your SSH configuration</a>.</p>
+<p><code class="varname">$USER</code> is an environment variable that is specific to Unix-like operating systems (e.g. Haiku, Linux, FreeBSD, etc.). If your system does not fall under that category, you will probably have to replace it.</p>
 
-The `<login>@` is only needed if your currently logged in username doesn't match your `review.haiku-os.org` username.
+<h4>Build Tools:</h4>
 
 ```sh
-git clone "ssh://<login>@git.haiku-os.org/buildtools" && scp -p <login>@git.haiku-os.org:hooks/commit-msg "buildtools/.git/hooks/"
+git clone "ssh://$USER@git.haiku-os.org/buildtools" && scp -p $USER@git.haiku-os.org:hooks/commit-msg "buildtools/.git/hooks/"
 ```
 
 <h4>Haiku:</h4>
 
-The `<login>@` is only needed if your currently logged in username doesn't match your `review.haiku-os.org` username.
-
 ```sh
-git clone "ssh://<login>@git.haiku-os.org/haiku"
-scp -p <login>@git.haiku-os.org:hooks/commit-msg "haiku/.git/hooks/"
+git clone "ssh://$USER@git.haiku-os.org/haiku" && scp -p $USER@git.haiku-os.org:hooks/commit-msg "haiku/.git/hooks/"
 ```
 
-<h4>Switching from read-only to write access</h4>
```

Comment:
this section was clearly labelled and this title made sense if following the previous paragraphs.

Now this info is removed, and there is no context for the commands below.

Reply:
This was accidental.

Reply:
Wait, no, "write access" insinuates "commit access", which is why I corrected that header and replaced it with `<h4>Sending in your first patch</h4>`.